### DISCUSSION TITLE
Is migration to jsbundling-rails / shakapacker necessary after upgrading to Rails 7?

### DISCUSSION THREAD

**Discussion by kell-maresh** [rails/jsbundling-rails#97] [General]
Having upgraded to rails version 7 for an existing project that's using `webpacker`. Is it necessary to migrate to `jsbundling-rails` / `shakapacker`?

Or can we continue to use `webpacker` (Version 5) until we find the bandwidth to make the jump to import maps / jsbundling (within a year perhaps)?

Would it be better to continue using rails version 6 with `webpacker` (Version 5) until we're ready to upgrade to rails 7 with import maps / jsbundling?

**Comment by brenogazzola** [comment]
If everything is still working, sure, go ahead and keep using Webpacker V5. The downsides are that you won't get any new features since Webpacker is locked to a specific version of Webpack, and that if you stumble into a bug, it's not going to be fixed. If I'm not mistaken, Shakapacker is a fork of Webpacker V6 and should be a much simpler migration.
